### PR TITLE
Exterior window look up prm routine

### PR DESCRIPTION
**Pull request by weilixu**
Updates to resolve the search criteria differences in the Exterior Window Look up in the PRM routine (2004 was resolved previously).

Also update code format after running rubocop.

### CODE REVIEW

File: lib/openstudio-standards/version.rb [hunk 1]
```diff
@@ -14,4 +14,4 @@ def self.git_revision
     return 'git-not-found-on-this-system'
   end
   VERSION = '0.2.15'.freeze
-end
\ No newline at end of file
+end
```
Comment (lymereJ) [inline lib/openstudio-standards/version.rb]: We typically ignore the autocorrect changes made by rubocop to `version.rb`, `Weather.Model.rb`, and `Weather.stat_file.rb`. The diff shown in this PR for these files seem reasonable, @mdahlhausen - could we accept these changes? Do you recall the reason why we usually exclude these changes?

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb [hunk 2]
```diff
@@ -137,4 +137,40 @@ def model_prm_skylight_to_roof_ratio_limit(model)
...
+          wwr_range['maximum_percent_of_surface'] = 30
+        elsif wwr_info[wwr_building_type] <= 40
+          wwr_range['minimum_percent_of_surface'] = 30.1
+          wwr_range['maximum_percent_of_surface'] = 40
+        else
+          wwr_range['minimum_percent_of_surface'] = nil
+          wwr_range['maximum_percent_of_surface'] = nil
+        end
+      else
+        OpenStudio.logFree(OpenStudio::Warn, 'openstudio.model.Model', 'No wwr_building_type found for ExteriorWindow or GlassDoor')
```
Comment (lymereJ) [inline lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb:171]: I would perhaps use a `OpenStudio::Error`, if the appropriate WWR isn't specified, the baseline fenestration performance won't be correct. If feel like a warning might be easily missed.

**Review by lymereJ — APPROVED** [review]
Looks good~ Just a two minor comments.

**Review by dmaddoxwhite — APPROVED** [review]
I'm deferring to Jeremy's review on this one.

File: lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb (hunk 2 above)
Comment (weilixu) [reply lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm_2019/ashrae_90_1_prm_2019.Model.rb]: Changed Warn to Error

File: lib/openstudio-standards/version.rb (hunk 1 above)
Comment (mdahlhausen) [reply lib/openstudio-standards/version.rb]: @lymereJ the rubocop edits are fine.  I'm surprised some method/file endings got changed - maybe worth a doublecheck that we have what rubocop wants.

**Pull request merged**